### PR TITLE
Update geom.go

### PR DESCRIPTION
**Pull request by 200sc**
Added public access to points and lengths back from dhconnely to the patrick-higgins fork.

Either this, or making p and q exposed publicly, are very helpful for a lot of use cases of the rtree structure.

### CODE REVIEW

File: rtree.go [hunk 1]
```diff
@@ -149,12 +154,16 @@ func (tree *Rtree) chooseNode(n *node, e entry, level int) *node {
...
+	//START:
 	if n == tree.root {
 		return n, nn
 	}
 
 	// Re-size the bounding box of n to account for lower-level changes.
 	en := n.getEntry()
+	// if en == nil {
+	// 	goto START
+	// }
```
Comment (patrick-higgins) [inline rtree.go:166]: Why are you leaving commented-out blocks in the code?

File: rtree.go [hunk 2]
```diff
@@ -296,13 +308,17 @@ func (n *node) pickSeeds() (int, int) {
 	left, right := 0, 1
 	maxWastedSpace := -1.0
 	var bb Rect
-	for i, e1 := range n.entries {
-		for j, e2 := range n.entries[i+1:] {
+	// for i := 0; i < len(n.entries)-1; i++ {
+	// 	e1 := n.entries[i]
```
Comment (patrick-higgins) [inline rtree.go:312]: Delete.

File: geom.go [hunk 3]
```diff
@@ -106,6 +106,16 @@ type Rect struct {
 	p, q Point // Enforced by NewRect: p[i] <= q[i] for all i.
 }
 
+// The coordinate of the point of the rectangle at i
+func (r *Rect) PointCoord(i int) float64 {
+	return r.p[i]
+}
+
+// The coordinate of the lengths of the rectangle at i
```
Comment (patrick-higgins) [inline geom.go:114]: Is this really a coordinate of lengths? Isn't it just a Length?

**Comment by 200sc** [comment]
Sorry, I figured this was ignored and have been modifying my branch for some experimental asynchronous things without regard for this pr. The comments are there as a history for myself of the original impl.

**Pull request closed without merging**